CONTRIBUTING.rst

```diff
@@ -0,0 +1,345 @@
+  asked to close the issue to confirm).
+
+
+
+Contributing code
+#################
+
+
+Code contributions can be submitted through github pull requests.
+
+
+Prior to contribute code
+************************
+
+Consider the following:
+
+- Pull requests for bug fixes or new features should be based on the
+  `master` branch.
+
+- The following github documentation may be of use:
+
+  - See `Using Pull Requests
+    <https://help.github.com/articles/using-pull-requests>`_
+    for more information about Pull Requests.
+
+  - See `Fork A Repo <http://help.github.com/forking/>`_ for an
+    introduction to forking a repository.
+
+  - See `Creating branches
+    <https://help.github.com/articles/creating-and-deleting-branches-within-your-repository/>`_
+    for an introduction on branching within GitHub.
+
+- Naming your branch: TODO
+
+- Ensure that you are the original author of your changes,
+  and if that is not the case, ensure that the borrowed/adapted code
+  is compatible with the SCT MIT license.
+
+  Keep in mind that you are responsible for your contributions!
+
+
+When submitting your pull request
+*********************************
+
+Consider the following guidelines:
+
+
+- PR title:
+
+  - Provide a concise and self-descriptive title.
+
+    Some terminology tips:
+
+    - When adding something, “add ...”
+
+    - When correcting a blatant issue, use “fixup ...”
+
+  - You may “scope” the title using the applicable command name(s),
+    folder or other "module" as a prefix.
+
+    Examples:
+
+    - “sct_testing: add ability to run tests in parallel”
+    - “documentation: sphinx: add a section about support”
+
+  - Do not include the applicable issue number(s) in the title.
+
+  The PR title is used to automatically generate the `Changelog
+  <https://github.com/neuropoly/spinalcordtoolbox/blob/master/CHANGES.md>`_
+  for each new release.
+
+- PR text:
+
+  - If the PR relates to open issue(s) don't forget to indicate that you are
+    fixing them, referring to their number in the PR introduction
+    (eg. “This PR brings a solution for #1234”).
+
+  - Explain the benefit of merging the PR.
+
+  - Explain the approach and possible drawbacks.
+
+    It does not hurt to duplicate/rephrase text coming from the PR commit messages.
+
+
+- PR commit titles (review the “commits” tab):
+
+  - Provide a concise and self-descriptive title (especially avoid > 80 characters)
+
+    Some terminology tips:
+
+    - When adding/moving/removing something, use “add/move/remove ...”
+
+    - When correcting a blatant issue, use “fixup ...”
+
+    - When refactoring code, use “refactor ...”
+
+  - You may “scope” the title using the applicable command name(s),
+    folder or other "module" as a prefix.
+
+  - If a commit is responsible for fixing an issue, post-fix the
+    description with ``(fixes #ISSUE_NUMBER)``.
+
+  - Examples:
+
+    Good:
+
+    - “sct_testing: add ability to run tests in parallel (fixes #1539)”
```

Comment:
another suggestion to ease the reading would be to alternate the font/format type when showing examples of PR/commit titles, as done [here](https://github.com/neuropoly/spinalcordtoolbox/blob/80e925f442da8154675e160bbfa7ae73d0cbf521/CONTRIBUTING.md#title).

Reply:
Changed citations to italic.